aea/aea.py

```diff
@@ -236,8 +237,8 @@ def _get_msg_and_handlers_for_envelope(
 
         if error_handler is None:
             self.logger.warning("ErrorHandler not initialized. Stopping AEA!")
```

Comment:
did not get.
change warning to exception? but there is no exception raised at the moment, so error is more suitable.
if raise an exception - it's raised on next line.

Reply:
Put differently; why do we first log the warning and then `raise StopRuntime` without a message. Could we not simply do `raise StopRuntime("ErrorHandler not initialized. Stopping AEA!")`?

Reply:
StopRuntime is for internal use, not to be logged as other exceptions.
it's used to avoid call runtime.stop from internal sync methods executed under runtime control.